spring-cloud-data-rest/pom.xml

```diff
@@ -80,6 +82,25 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+				<configuration>
+					<classifier>exec</classifier>
```

Comment:
Agreed we will need the thin JAR in the module case. Here (and probably the spring-cloud-stream-module-launcher as well) it would be better to have *the* JAR be the bootified one. It does make this trickier to avoid hardcoding a version in here (while the wildcard-match on *-exec made that possible), but I don't think that's enough of a reason to force a qualifier on users who simply want to launch the AdminApplication as a boot app.